R/readpolicy_general.R

```diff
@@ -0,0 +1,46 @@
+#' readpolicy_general
+#'
+#' @param initial belief to compute dot product with alpha vectors.
+#' @param fs the initial fully observable state
```

Comment:
just need to add something to the documentation to indicate that `fs` and `Num_fs` are optional, and when they might be used.